woocommerce/woocommerce-android
# Fix Login Prologue UI issues

Summary
==========
Fix issue #8727 by tweaking the login prologue view to correctly resize the description text until it's no longer possible to display it. Allowing a proper UI placement without truncating elements. It also updates the Circle White button dark mode color to follow the design review suggestions. ref C03L1NF1EA3-slack-p1681297109881089

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| <img alt="Screenshot 2023-04-17 at 21 31 51" src="https://user-images.githubusercontent.com/5920403/232638848-094aafaf-e952-468a-95c9-e92165b7b36b.png">| <img alt="Screenshot 2023-04-17 at 21 28 01" src="https://user-images.githubusercontent.com/5920403/232638658-7b39e3bb-a1b0-4d5f-a99f-96e5dce00a68.png"> |

| Small display  | Mid sized display | Large display |
| ------------- | ------------- | ------------- |
| <img alt="Screenshot 2023-04-17 at 21 24 31" src="https://user-images.githubusercontent.com/5920403/232638602-79699780-404b-4519-84ba-345b9c8cad99.png"> | <img alt="Screenshot 2023-04-17 at 21 24 43" src="https://user-images.githubusercontent.com/5920403/232638625-6a9229d8-6fe1-40e7-9fa1-d470750a845f.png"> | <img alt="Screenshot 2023-04-17 at 21 25 04" src="https://user-images.githubusercontent.com/5920403/232638646-907954f3-4e4e-4b5a-a9e6-ee7eacbdd99d.png"> | 

How to Test
==========
1. Simply test the Login Prologue in multiple screen sizes and with dark mode. Verify that the UI placement is adequate and adjusts itself by adapting the `description` text until it's no longer displayable.

Update release notes:
